src/activities/solar_system/QuizScreen.qml

```diff
@@ -0,0 +1,295 @@
+/* GCompris - QuizScreen.qml
+ *
+ * Copyright (C) 2018 Aman Kumar Gupta <gupta2140@gmail.com>
+ *
+ * Authors:
+ *   Aman Kumar Gupta <gupta2140@gmail.com>
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.6
+import GCompris 1.0
+import QtGraphicalEffects 1.0
+import QtQuick.Controls 1.5
+
+import "../../core"
+import "solar_system.js" as Activity
+
+Item {
+    id: mainQuizScreen
+    width: parent.width
+    height: parent.height
+    focus: true
+
+    property alias score: score
+    property alias optionListModel: optionListModel
+    property alias restartAssessmentMessage: restartAssessmentMessage
+    property string planetRealImage
+    property string question
+    property string closenessMeterValue
+    property int numberOfCorrectAnswers: 0
+
+    Rectangle {
+        id: questionArea
+        anchors.right: score.left
+        anchors.top: parent.top
+        anchors.left: parent.left
+        anchors.margins: 10 * ApplicationInfo.ratio
+        height: questionText.height + 10 * ApplicationInfo.ratio
+        color: 'white'
+        radius: 10
+        border.width: 3
+        opacity: 0.8
+        border.color: "black"
+        GCText {
+            id: questionText
+            horizontalAlignment: Text.AlignHCenter
+            verticalAlignment: Text.AlignVCenter
+            anchors.centerIn: parent.Center
+            color: "black"
+            width: parent.width
+            wrapMode: Text.Wrap
+            text: mainQuizScreen.question
+        }
+    }
+
+    // Model of options for a question
+    ListModel {
+        id: optionListModel
+    }
+
+    // This grid has image of the planet in its first column/row (row in case of vertical screen) and the options on the 2nd column/row
+    Grid {
+        id: imageAndOptionGrid
+        columns: (background.horizontalLayout && !items.assessmentMode && items.bar.level != 2) ? 2 : 1
+        spacing: 10 * ApplicationInfo.ratio
+        anchors.top: (questionArea.y + questionArea.height) > (score.y + score.height) ? questionArea.bottom : score.bottom
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.margins: 10 * ApplicationInfo.ratio
```

Comment:
can you add topMargin to 0? Else on some ratios, the latest answer overrides the closeness bar